recipe_scrapers/momswithcrockpots.py

```diff
@@ -34,13 +34,3 @@ def instructions(self):
         return "\n".join(
             [normalize_string(instruction.get_text()) for instruction in instructions]
         )
-
-    def ratings(self):
-        return round(
-            float(
-                self.soup.find(
-                    "span", {"class": "wprm-recipe-rating-average"}
-                ).get_text()
-            ),
-            2,
-        )
```

Comment:
Nice - using the `schema.org` default field retrieval here instead of a custom implementation :+1: 

Maybe we could add code/testing/process to check for cases where scrapers have unnecessary custom scraping code -- i.e. `schema.org` field returns the same as the method call, but the method call is an override?  (cc #617)

Reply:
(added a checklist item for this feature to the list in the description of #617)